### ISSUE TITLE
Experimental stringification has a problem with mixed quotes in some instanaces

### ISSUE THREAD

**Issue by underscorebrody** [ericclemmons/grunt-angular-templates#149]
Example of a problem scenario:
```
<input name="password" class="form-control" data-ng-model="admin_password" type="password" data-ng-attr-placeholder="{{'Enter a password for the user \'admin\'' | translate}}" required>
```

**Comment by underscorebrody** [comment]
cc @evilaliv3

**Comment by evilaliv3** [comment]
this issue has been already solved by #146 and does not sussist anymore.

thanks @underscorebrody

**Comment by underscorebrody** [comment]
Okay thanks!
